app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/PlanWizardInstancePropertiesStepTable.js

```diff
@@ -0,0 +1,427 @@
+import React from 'react';
```

Comment:
Codeclimate has ruined this file, can't even review it 😭 😡

Reply:
😢

Reply:
🤗